pkg/option/config.go

```diff
@@ -67,6 +67,8 @@ type config struct {
 	MemProfile string
 	PprofAddr  string
 
+	EventsQueueSize int
```

Comment:
This seems too similar of a name to EventQueueSize, how about something like `PerfEventsQueueSize`?